lib/regexp/regexp.go

```diff
@@ -0,0 +1,258 @@
+// Copyright 2021 The Bazel Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Package regexp provides regular expression related functions.
```

Comment:
missing hyphen

Better, put the emphasis at the end: "functions related to regular expressions".

We should state that it uses RE2 specifically, as that implies certain bounds on running time and certain restrictions on features.